tests/data/minitest/record_test_result.json

```diff
@@ -22,6 +26,10 @@
         {
           "type": "testcase",
           "name": "test_should_not_save_user_without_name"
+        },
+        {
+          "type": "class",
+          "name": "UserCopyTest"
```

Comment:
it seems result json order is still file -> testcase -> class

Reply:
Is it correct??

Reply:
Yes, it is.

Reply:
It's because `assert_json_orderless_equal` is used at  https://github.com/launchableinc/cli/blob/main/tests/cli_test_case.py#L239.

Reply:
To test the order, I've added https://github.com/launchableinc/cli/pull/846/files#diff-c85217de42a62611db47dc8b7f9c6a9efc83f793379b372b3b1c65dd772a2f68R23-R43.

Reply:
I see but I think it's my bad https://github.com/launchableinc/cli/pull/817

Because the test paths should be ordered.
we (Launchable) cannot allow miss ordering e.g.) `file -> testcase -> class`.

You don't have to fix `assert_json_orderless_equal` now. But could you update this result JSON to the correct order to prevent misunderstanding?

Reply:
Let me fix it in another PR.

Reply:
`assert_json_orderless_equal` -> another PR
chunk_json files -> this PR

OK?

Reply:
I plan to fix both changes in another PR since we can check if it works.

Reply:
OK